include/common/set.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef LABWC_SET_H
+#define LABWC_SET_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define LAB_SET_MAX_SIZE 16
+
+struct lab_set {
+	uint32_t values[LAB_SET_MAX_SIZE];
+	int size;
+};
+
+bool lab_set_exists(struct lab_set *set, uint32_t value);
```

Comment:
I like the new `set` utilities, nice work. I'd suggest naming this `lab_set_contains` as "contains" is the more typical name for this method.

Reply:
Good idea. Fixed.